notebooks/shield-dash-import.py

```diff
@@ -0,0 +1,93 @@
+import datetime
+from operator import itemgetter
+
+import ujson
+from moztelemetry import get_pings
+
+
+PING_NAME = 'x-shield-studies'
+HEARTBEAT_NAME = 'x-shield-study-performance-1'
+STUDY_NAME = 'screen Performance X1'
+STUDY_START = '20160325'
+TODAY = datetime.date.today().strftime('%Y%m%d')
+
+
+def getShieldProps(p):
+    out = {
+        'client_id': p.get('clientId'),
+        'channel': p['application']['channel'],
+        'creation_date': datetime.datetime.utcfromtimestamp(
+            long(p['meta']['creationTimestamp']) // 1e9),
+        'doc_type': p['meta']['docType']
+    }
+    for k in ['firstrun', 'msg', 'name', 'variation']:
+        if k == 'firstrun':
+            out[k] = datetime.datetime.utcfromtimestamp(
+                int(p['payload'][k]) // 1e3)
+        else:
+            out[k] = p['payload'][k]
+    return out
```

Comment:
@jezdez I tried to make the date math a bit nicer here. Let me know what you think.